yellowbrick/features/base.py

```diff
@@ -309,3 +312,130 @@ def _determine_target_color_type(self, y):
                 "could not determine target color type "
                 "from target='{}' to '{}'"
             ).format(self.target_type, self._target_color_type))
+
+
+##########################################################################
+## Projection Visualizers
+##########################################################################
+
+class ProjectionVisualizer(DataVisualizer):
```

Comment:
Perhaps we should put this in `yellowbrick/features/projection.py`? `base.py` is getting a little full at this point!

Reply:
Also to wrap this up, I'd like to make sure we have a complete docstring that describes what this base class does for other yellowbrick developers.

Reply:
I will write the docstring and move it to `projection.py`!

Reply:
As I'm in the airport I'm having trouble with my Internet connection and can't easily check the source code - is there already a `projection.py`? (I'm hoping there is not).

Reply:
No, currently we do not have projection.py!

Reply:
ok awesome, I'm now thinking about the following organization:

```
yellowbrick
└── features
|   ├── __init__.py
|   └── projection
|   |   ├── __init__.py
|   |   ├── base.py
|   |   ├── manifold.py
|   |   ├── pca.py
|   ├── base.py
|   ├── decomposition.py
```

But let me think about this a little more.